src/registrar/templates/includes/domain_request_status_manage.html

```diff
@@ -173,7 +173,7 @@ <h2 class="text-primary-darker"> Summary of your domain request </h2>
             {% endwith %}
         {% endif %}
 
-        {% if DomainRequest.organization_name %}
+        {% if DomainRequest.converted_organization_name %}
```

Comment:
@asaki222 isn't this on the request form? Or am I reading this wrong? For your ticket the idea is that you will only change the ones for django admin, not every domain request form peice.

Reply:
@abroddrick I didn't get a chance to comb through it(this is a rough draft), and clean it up yet. I was just focusing on getting a general working solution.

Reply:
@abroddrick Removed!